src/components/controller/loader.ts

```diff
@@ -1,19 +1,25 @@
+interface LoaderOptions {
```

Comment:
Better to rename it to `ILoaderOptions`, it will be better for readability to start all interfaces with 'I'